docker-compose.yml

```
@@ -61,47 +61,46 @@ services:
-      - OVERRIDE_DATABASE_HOSTNAME=${OVERRIDE_DATABASE_HOSTNAME:-}
-      - OVERRIDE_DATABASE_PASSWORD=${OVERRIDE_DATABASE_PASSWORD:-}
-      - OVERRIDE_DATABASE_PORT=${OVERRIDE_DATABASE_PORT:-}
-      - OVERRIDE_DATABASE_USERNAME=${OVERRIDE_DATABASE_USERNAME:-}
-      - PORT=${PORT:-8080}
-      - DATABASE_NAME=${DATABASE_NAME}
-      - DATABASE_HOSTNAME=${DATABASE_HOSTNAME}
-      - DATABASE_PASSWORD=${DATABASE_PASSWORD}
-      - DATABASE_PORT=${DATABASE_PORT}
-      - DATABASE_USERNAME=${DATABASE_USERNAME}
-      - QUOTES_DOMAIN=${QUOTES_DOMAIN}
-      - REQUIRE_HTTPS=${REQUIRE_HTTPS:-}
-      - S3_ACCESS_KEY_ID=${S3_ACCESS_KEY_ID}
-      - S3_BUCKET=${S3_BUCKET}
-      - S3_ENDPOINT=${S3_ENDPOINT}
-      - S3_SECRET_ACCESS_KEY=${S3_SECRET_ACCESS_KEY}
-      - SES_ACCESS_KEY_ID=${SES_ACCESS_KEY_ID}
-      - SES_AWS_REGION=${SES_AWS_REGION}
-      - SES_SECRET_ACCESS_KEY=${SES_SECRET_ACCESS_KEY}
-      - SESSION_SECRET=${SESSION_SECRET:-secret}
-      - SUPPORT_EMAIL=${SUPPORT_EMAIL}
-      - WHITELISTED_EMAILS=${WHITELISTED_EMAILS}
+      APP_PATH: ${APP_PATH:-/index.html}
+      AUTH_METHOD_OVERRIDE: ${AUTH_METHOD_OVERRIDE:-}
+      CLOUDFRONT_KEY_PAIR_ID: ${CLOUDFRONT_KEY_PAIR_ID:-}
+      CLOUDFRONT_PRIVATE_KEY: ${CLOUDFRONT_PRIVATE_KEY:-}
+      DATABASE_HOSTNAME: mysql
+      DATABASE_NAME: ${DATABASE_NAME:-ToadReader}
+      DATABASE_PASSWORD: ${DATABASE_PASSWORD}
+      DATABASE_PORT: ${DATABASE_PORT:-3306}
+      DATABASE_USERNAME: ${DATABASE_USERNAME}
+      DEBUG: ${DEBUG:-}
+      DEFAULT_DOMAIN_TO_QUERY_FOR_EMAIL: ${DEFAULT_DOMAIN_TO_QUERY_FOR_EMAIL}
+      DEFAULT_FROM_EMAIL: ${DEFAULT_FROM_EMAIL}
+      DEFAULT_IDP_DOMAIN: ${DEFAULT_IDP_DOMAIN:-}
+      DEV_NETWORK_IP: ${DEV_NETWORK_IP:-localhost}
+      IPV4_INTERNAL_NETWORK_REGEX: ${IPV4_INTERNAL_NETWORK_REGEX:-}
+      IPV6_INTERNAL_NETWORK_REGEX: ${IPV6_INTERNAL_NETWORK_REGEX:-}
+      IS_DEV: ${IS_DEV-1}
+      LOG_LEVEL: ${LOG_LEVEL}
+      LOGIN_TEST_CODE: ${LOGIN_TEST_CODE:-}
+      LOGIN_TEST_EMAIL: ${LOGIN_TEST_EMAIL:-}
+      MARKETING_DOMAIN: ${MARKETING_DOMAIN}
+      OVERRIDE_DATABASE_HOSTNAME: ${OVERRIDE_DATABASE_HOSTNAME:-}
+      OVERRIDE_DATABASE_NAME: ${OVERRIDE_DATABASE_NAME:-}
+      OVERRIDE_DATABASE_PASSWORD: ${OVERRIDE_DATABASE_PASSWORD:-}
+      OVERRIDE_DATABASE_PORT: ${OVERRIDE_DATABASE_PORT:-}
+      OVERRIDE_DATABASE_USERNAME: ${OVERRIDE_DATABASE_USERNAME:-}
+      PORT: ${PORT:-8080}
+      QUOTES_DOMAIN: ${QUOTES_DOMAIN}
+      REQUIRE_HTTPS: ${REQUIRE_HTTPS:-}
+      S3_ACCESS_KEY_ID: ${S3_ACCESS_KEY_ID:-minio}
+      S3_BUCKET: ${S3_BUCKET:-toadreader}
+      S3_ENDPOINT: ${S3_ENDPOINT:-http://minio:9000}
+      S3_SECRET_ACCESS_KEY: ${S3_SECRET_ACCESS_KEY:-minio123}
+      SES_ACCESS_KEY_ID: ${SES_ACCESS_KEY_ID}
+      SES_AWS_REGION: ${SES_AWS_REGION}
+      SES_SECRET_ACCESS_KEY: ${SES_SECRET_ACCESS_KEY}
+      SESSION_SECRET: ${SESSION_SECRET:-secret}
+      SUPPORT_EMAIL: ${SUPPORT_EMAIL}
+      USE_DEVELOPMENT_S3: ${USE_DEVELOPMENT_S3-1}
```

Looks like you're missing a colon:
```suggestion
      USE_DEVELOPMENT_S3: ${USE_DEVELOPMENT_S3:-1}
```

---

No it is [defaulting while allowing an empty replacement](https://docs.docker.com/compose/how-tos/environment-variables/variable-interpolation/#interpolation-syntax)